modules/ROOT/pages/x12-receive-read-settings.adoc

```diff
@@ -4,7 +4,7 @@ Partner Manager enables you to configure the X12 acknowledgments, validation rul
 
 == Configure the Receive Settings
 
-. In the left navigation menu, select the partner for whom to configure the send settings.
+. In the left navigation menu, select the partner for whom you want to configure the settings.
```

Comment:
You specify "menu" here, which seems correct. When you use "the" it applies to a noun/object. So I would think we need the word menu.

Reply:
This is one of the sections that will likely be rewritten - I am going to leave this here for now.